src/python/WMComponent/DBS3Buffer/MySQL/NewSubscription.py

```diff
@@ -40,15 +40,17 @@ def _createPhEDExSubBinds(self, datasetID, subscriptionInfo, custodialFlag):
 
         binds = []
         for site in sites:
-            binds.append({'id': datasetID,
-                          'site': site,
-                          'custodial': custodialFlag,
-                          'auto_approve': 1 if site in subscriptionInfo['AutoApproveSites'] else 0,
-                          'move': isMove,
-                          'priority': subscriptionInfo['Priority'],
-                          'phedex_group': phedex_group,
-                          'delete_blocks': delete_blocks,
-                          'dataset_lifetime': subscriptionInfo['DatasetLifetime']})
+            bind = {'id': datasetID,
+                    'site': site,
+                    'custodial': 0 if custodialFlag is None else custodialFlag,
+                    'auto_approve': 1 if site in subscriptionInfo['AutoApproveSites'] else 0,
+                    'move': 0 if isMove is None else isMove,
```

Comment:
isMove requires validation too, someone can pass `-1` and code will not spot it.

Reply:
I'm actually removing this check on `isMove` because it's already validated a few lines above.